app/components/dashboard/replication_status_component.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="card border-0 text-start">
+  <div class="card-body" />
+    <div class="card-title fs-4 fw-semibold d-flex justify-content-between">
+      Replication Zips
+      <span class="badge rounded-pill <%= replication_and_zip_parts_ok? ? 'bg-success' : 'bg-danger' %>">
+        <%= replication_and_zip_parts_ok? ? 'OK' : 'Error' %>
+      </span>
+    </div>
+    <ul class="list-group border border-2 border-secondary">
+      <li class="list-group-item d-flex justify-content-between align-items-center">
+        <h6 class="d-flex justify-content-start">ZipPart Statuses</h6>
+        <span class="badge rounded-pill <%= zip_parts_ok? ? 'bg-success' : 'bg-danger' %>">
+          <%= zip_parts_ok? ? 'OK' : 'Error' %>
+        </span>
+      </li>
+      <% endpoint_data.each do |endpoint_name, endpoint_info| %>
+        <% replication_count = endpoint_info[:replication_count] %>
+        <% endpoint_replication_ok = replication_count == num_object_versions_per_preserved_object %>
+        <li class="list-group-item d-flex justify-content-between align-items-center">
+          <h6 class="d-flex justify-content-start">
+            <span class="text-secondary">Endpoint:&nbsp;</span>
+            <%= endpoint_name %>&nbsp;&nbsp;
+          </h6>
+          <span class="badge rounded-pill <%= endpoint_replication_ok ? 'bg-success' : 'bg-danger' %>">
+            <%= endpoint_replication_ok ? 'OK' : 'Error' %>
+          </span>
+        </li>
+      <% end %>
+      <li class="list-group-item d-flex justify-content-between align-items-center">
+        <h6><a href="resque">Redis queues</a></h6>
```

Comment:
Since we're in the process of putting in Sidekiq, can we not create more references to Resque?

Reply:
I think the link is useful;  I named it "redis queue" and figure we'll update it once sidekiq goes in.  Not sure how soon that will be.  It's an easy update here, though!

Reply:
Yes, but there is already a sidekiq PR. Now someone needs to remember to keep them in sync. I'd prefer to see adding this as a ticket rather then set-up a guaranteed conflict.